bin/metrics.js

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var fs = require('fs')
+var section
+
+module.exports = {
+  run: run,
+  countAccounts: countAccounts,
+  countVerifiedAccounts: countVerifiedAccounts,
+  countAccountsWithTwoOrMoreDevices: countAccountsWithTwoOrMoreDevices,
+  countAccountsWithThreeOrMoreDevices: countAccountsWithThreeOrMoreDevices,
+  countAccountsWithMobileDevice: countAccountsWithMobileDevice
+}
+
+if (require.main === module) {
+  module.exports.run(
```

Comment:
Calling this in context so that `run` can call the query functions on `this`. Doing that makes it easier to mock them in the unit tests.